docs/Makefile

```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I don't see how this is related to your PR. Please drop the chance. If you want to build the docs, please use `python setup.py build_sphinx`

Reply:
how this is related to a documentation PR … without the Makefile, and without attempting `python setup.py build_sphinx`, it was unpractical to suggest corrections in the documentation. some are purely typesetting, and wanted to make sure that sphinx understood the formatting, because part of the output was a bit out of sync.
do you insist, in not having the `Makefile` in the docs? it doesn't do any harm, does it?  anyway, I will double check that the choice is properly documented ("if you want to build the docs, please use `python setup.py build_sphinx`).